api/stats/kickoff.proto

```diff
@@ -51,14 +51,21 @@ enum TouchPosition {
   UNKNOWN_TOUCH_POS=5;
 };
 
+message PlayerPositionData {
+    optional float pos_x = 1;
+    optional float pos_y = 2;
+    optional float pos_z = 3;
+}
+
 message KickoffPlayer {
 optional api.PlayerId player = 1;
 optional KickoffPosition kpos = 2;
 optional TouchPosition tpos = 3;
```

Comment:
if all of these are xyz it may make more sense to make a general position object to fill these with

Reply:
kpos/tpos are the enum fields for BALL/OFFCENTER, etc.

there is events/BallData that has the same members, I could just use that

Reply:
Okay lets use this instead as this will be more readable.  I do not think we need to combine those.  It does not take anymore space and makes it more readable.

Also we should probably change the field names to kickoff_position and touch_position and player_position so it is more readable in code.

Reply:
I'll swap those around, is _position alright for the enums?  I was thinking _location might be better because it's not x/y/z